.github/workflows/elixir.yml

```diff
@@ -4,18 +4,25 @@ on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     container:
-      image: elixir:1.9.1-slim
+      image: elixir:1.10.2-slim
 
     steps:
     - uses: actions/checkout@v1
+    - uses: actions/cache@v1
+      with:
+        path: _build
+        key: build-${{ hashFiles('mix.lock') }}
```

Comment:
Great! there is a catching as well. or else it could take forever 👍 

But I wonder, should we just specify a single key to share the caching to all builds?

Reply:
@teerawat1992

Reply:
Yeah, i think we can na 👍

Reply:
According to the recommendation, i'll cache with elixir+eralng version instead 🙇 
https://github.com/jeremyjh/dialyxir#continuous-integration